tsup.config.ts

```diff
@@ -13,10 +13,9 @@ export default defineConfig({
     cloudflare: "./platforms/cloudflare.ts",
   },
   format: ["cjs", "esm"],
-  splitting: true,
```

Comment:
because we removed the explicit setting, code splitting is not applied to CJS

<img width="632" alt="image" src="https://github.com/user-attachments/assets/11c74767-f51b-4d2a-b2e0-c6bc00899c50">

I think because of this reason there is an increase in the package size from [379](https://www.npmjs.com/package/@upstash/qstash/v/2.7.10?activeTab=code) to [867](https://www.npmjs.com/package/@upstash/qstash/v/2.7.11?activeTab=code) kB

old

<img width="946" alt="image" src="https://github.com/user-attachments/assets/144fdb4f-c126-4d23-ab47-e2eda6c50413">

new

<img width="946" alt="image" src="https://github.com/user-attachments/assets/ff839452-d399-4e46-93a8-02b81e310a4f">

let's keep this in mind, we can revert if necessary

Reply:
Updated the pr and release description, thanks.